CODING.md

```diff
@@ -5,6 +5,7 @@
 ### Node.js
 
 iTowns uses NPM as the build tool. If you haven't already, install Node.js: https://nodejs.org/
+As a minimum you must have Node.js 10 or superior to develop in iTowns.
```

Comment:
Missed that one earlier: same "At a minimum" vs "or superior" (and "At" a minimum rather than "As")

Reply:
yes